docker/databricks/00-custom-spark-driver-defaults.conf

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+[driver] {
```

Comment:
the only different in this file and alluxio one is the setting:
 "spark.rapids.alluxio.automount.enabled" = "true". ?

perhaps we can handle that differently to not have to have 2 copies of the same configs

Reply:
Actually there are 2 configs:
```
 "spark.databricks.io.cache.enabled" = "false"
 "spark.rapids.alluxio.automount.enabled" = "true"
```

I'm guessing disabling the databricks io cache is required to enable alluxio, is that correct?